Topics/Software_Engineering/Deploying_Personal_Website.md

```diff
@@ -0,0 +1,92 @@
+# Deploying your personal website using GitHub
+
+### Why should you create a personal website?
```

Comment:
Have you considered adding one example of a static personal website and one for a dynamic personal website? I think that it would help the reader clearly visualize what the difference would be.

Reply:
Thanks for the suggestion - added 2 examples!